src/defines/define.ts

```diff
@@ -0,0 +1,10 @@
+import {MagickFormat} from '../magick-format';
+
+export interface IDefine {
+
+  format: MagickFormat;
+
+  name: string;
+
+  value: string;
```

Comment:
The Magick.NET code allowed a null value here. Thanks for making it not nullable here. Just fixed that in the Magick.NET repository.